cluster/crypto/Makefile

```diff
@@ -0,0 +1,56 @@
+
+.PHONY: all clean
+all: all.tar
+
+clean:
+	rm -f *.tar
+	rm -f *.pem
+	rm -f *.csr
+	rm -f ca.srl
+	rm -f openssl.cnf
+
+# config
+openssl.cnf: openssl.cnf.in san-extras
+	rm -f openssl.cnf
+	cat openssl.cnf.in >> openssl.cnf
+	cat san-extras >> openssl.cnf
+
+# root CA
+ca-key.pem:
```

Comment:
Nit: I lean toward `ca.key/ca.crt` vs `ca-key.pem/ca.pem`
 
Also, this could be more generic target to cover `ca-key` `apiserver-key` `kubelet-key`:

```
%-key.pem:
    openssl genrsa -out "$@" 2048
```